intel-mkl-tool: Compilation problem with anyhow 1.0.49

Following the release of anyhow 1.0.49, I get the following compilation error:
<pre>  
...
   Compiling ndarray-linalg v0.14.1
   Compiling zstd v0.5.4+zstd.1.4.7
   Compiling intel-mkl-tool v0.2.0+mkl2020.1
error[E0532]: expected tuple struct or tuple variant, found function `Ok`
  --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/intel-mkl-tool-0.2.0+mkl2020.1/src/entry.rs:75:16
   |
75 |         if let Ok(dir) = std::env::var("OUT_DIR") {
   |                ^^ not a tuple struct or tuple variant
   |
help: consider importing one of these items instead
   |
1  | use core::result::Result::Ok;
   |
1  | use std::result::Result::Ok;
   |
...
</pre>

It seems like the new `Ok()` function introduced in anyhow breaks the code. see [anyhow 1.0.49 release](https://github.com/dtolnay/anyhow/releases/tag/1.0.49). Pinning anyhow to 1.0.48 solves the problem.
@dtolnay, to also bring this to your attention.

--- a/intel-mkl-src/build.rs
+++ b/intel-mkl-src/build.rs
@@ -22,9 +22,9 @@
 
 #![cfg_attr(feature = "download", allow(unreachable_code))]
 
-use anyhow::*;
+use anyhow::{bail, Result};
 use intel_mkl_tool::*;
-use std::{env, path::*};
+use std::{env, path::PathBuf};
 
 #[cfg(feature = "mkl-static-lp64-iomp")]
 const MKL_CONFIG: &str = "mkl-static-lp64-iomp";

--- a/intel-mkl-tool/src/cli.rs
+++ b/intel-mkl-tool/src/cli.rs
@@ -1,4 +1,4 @@
-use anyhow::*;
+use anyhow::{bail, Result};
 use intel_mkl_tool::*;
 use std::{env, path::PathBuf};
 use structopt::StructOpt;

--- a/intel-mkl-tool/src/config.rs
+++ b/intel-mkl-tool/src/config.rs
@@ -1,5 +1,5 @@
 use crate::*;
-use derive_more::*;
+use derive_more::Display;
 
 pub const VALID_CONFIGS: &[&str] = &[
     "mkl-dynamic-ilp64-iomp",

--- a/intel-mkl-tool/src/lib.rs
+++ b/intel-mkl-tool/src/lib.rs
@@ -92,8 +92,8 @@
 
 #![cfg_attr(not(feature = "archive"), allow(dead_code))]
 
-use anyhow::*;
-use std::path::*;
+use anyhow::{bail, Result};
+use std::path::{Path, PathBuf};
 
 mod config;
 mod entry;

